jetstream/experimenter.py

```diff
@@ -32,12 +71,48 @@ class FeatureEventTelemetry:
     event_object: Optional[str]
     event_value: Optional[str]
 
+    def to_summaries(
+        self, feature_slug: str, experiment: ExperimentConfiguration
+    ) -> Iterable[Summary]:
+        clauses = [f"event_category = '{self.event_category}'"]
+        for k in ("method", "object", "value"):
+            if v := getattr(self, f"event_{k}"):
+                clauses.append(f"event_{k} = '{v}'")
+        predicate = " AND ".join(clauses)
+
+        kwargs: Dict[str, Any] = {}
+        if experiment.reference_branch:
+            kwargs["ref_branch_label"] = experiment.reference_branch
+
+        ever_used = Summary(
+            Metric(
+                f"{feature_slug}_ever_used",
+                mozanalysis.metrics.desktop.events,
+                f"COALESCE(COUNTIF({predicate}), 0) > 0",
+            ),
+            statistics.Binomial(**kwargs),
+        )
+
+        sum_metric = Metric(
+            f"{feature_slug}_sum",
+            mozanalysis.metrics.desktop.events,
+            f"COALESCE(COUNTIF({predicate}), 0)",
+        )
+
+        used_mean = Summary(sum_metric, statistics.BootstrapMean(**kwargs))
+        used_deciles = Summary(sum_metric, statistics.Deciles(**kwargs))
+
+        return [ever_used, used_mean, used_deciles]
+
 
 @attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
 class Feature:
     slug: str
     telemetry: Union[FeatureEventTelemetry, FeatureScalarTelemetry]
 
+    def to_summaries(self, experiment: ExperimentConfiguration) -> Iterable[Summary]:
```

Comment:
Type checking is more effective if we pass in the container instead of just the string since the container has an interesting type and a string is just a string; Python passes objects by reference and not value so there's no additional cost to passing in the larger object.